examples/md-flexible/src/Simulation.h

```diff
@@ -186,7 +186,7 @@ void Simulation<Particle, ParticleCell>::initialize(const MDFlexConfig &mdFlexCo
   autopas.setCutoff(_config->cutoff.value);
   autopas.setRelativeOptimumRange(_config->relativeOptimumRange.value);
   autopas.setMaxTuningPhasesWithoutTest(_config->maxTuningPhasesWithoutTest.value);
-  autopas.setTestsUntilFirstPrediction(_config->testsUntilFirstPrediction.value);
+  autopas.setEvidenceFirstPrediction(_config->evidenceForPrediction.value);
```

Comment:
I think the setter, member of `autopas.h` and the config entry should have the same name.